server/auth/types/authentication_type.ts

```diff
@@ -210,7 +210,8 @@ export abstract class AuthenticationType implements IAuthenticationType {
       }
     }
     if (!authInfo) {
-      authInfo = await this.securityClient.authinfo(request, authHeaders);
+      const authRequestType = cookie.isAnonymousAuth ? 'anonymous' : cookie.authType;
```

Comment:
Should probably standardize the authType string use to be a enum with constants.

Reply:
good catch. I've pushed a new commit by replacing this magic constant.